### PR TITLE
TG-56, TG-57 - add attendance list

### PR DESCRIPTION
**Pull request by mswianiewicz**

### CODE REVIEW

File: apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/attendance/api/AttendanceController.java [hunk 1]
```diff
@@ -27,6 +30,8 @@
 @Validated
 public class AttendanceController {
     private final AttendanceService attendanceService;
+    private final EventService eventService;
+    private final EventService meetingService;
```
Comment (Delebrith) [inline apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/attendance/api/AttendanceController.java:34]: Please remove redundant event service.

**Review by Delebrith — CHANGES_REQUESTED** [review]

**Review by jgajownik — DISMISSED** [review]
BE changes LGTM

**Review by FrozenPear42 — DISMISSED** [review]
Great work on frontend!

ap,550x550,12x16,1,transparent,t u7

File: apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/attendance/AttendanceServiceImpl.java [hunk 2]
```diff
@@ -50,17 +57,16 @@ public Attendance putIntoAttendanceList(Meeting meeting, User user) {
     public void markAsPresent(Long id, String token, Long meetingId) {
         Attendance attendance = attendanceRepository.findByIdAndTokenAndMeeting_Id(id, token, meetingId)
                 .orElseThrow(() -> new AttendanceNotFoundException(
-                        "Attendance not found",
-                        ImmutableMap.of("id", id, "token", token, "meetingId", meetingId)));
-        checkIfUserIsOrganizerOrSpeaker(attendance);
+                        "Attendance not found", ImmutableMap.of("id", id, "token", token)));
```
Comment (Delebrith) [inline apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/attendance/AttendanceServiceImpl.java:60]: Why is meetingId deleted?

**Review by Delebrith — COMMENTED** [review]
Just one thing that confused me, all the rest is OK

**Review by Delebrith — APPROVED** [review]
Good job 👍

File: apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/event/EventServiceImpl.java [hunk 3]
```diff
@@ -19,10 +19,7 @@
 import org.springframework.stereotype.Service;
 
 import javax.transaction.Transactional;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Map;
+import java.util.*;
```
Comment (SylviaLee89) [inline apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/event/EventServiceImpl.java:22]: our static analysis engine detects that your pull request includes a change from an explicit import to a wildcard import. According to the best practice (http://www.javadude.com/articles/importondemandisevil.html), import star (import on demand) is a bad practice, wildcard import can sometimes cause name conflicts and ambiguities.

File: apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/event/api/EventController.java [hunk 4]
```diff
@@ -1,13 +1,8 @@
...
 
-import edu.pw.apsienrollment.common.api.dto.PageRequestDTO;
 import edu.pw.apsienrollment.common.api.dto.SearchRequestDTO;
 import edu.pw.apsienrollment.event.EventService;
-import edu.pw.apsienrollment.event.api.dto.AllowedToCreateDto;
-import edu.pw.apsienrollment.event.api.dto.EventDto;
-import edu.pw.apsienrollment.event.api.dto.EventRequestDto;
-import edu.pw.apsienrollment.event.api.dto.EventWithMeetingsDto;
-import edu.pw.apsienrollment.event.api.dto.MeetingDto;
+import edu.pw.apsienrollment.event.api.dto.*;
```
Comment (SylviaLee89) [inline apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/event/api/EventController.java:5]: our static analysis engine detects that your pull request includes a change from an explicit import to a wildcard import. According to the best practice (http://www.javadude.com/articles/importondemandisevil.html), import star (import on demand) is a bad practice, wildcard import can sometimes cause name conflicts and ambiguities.

**Pull request merged**